compiler/optimizer/OMRSimplifierHelpers.cpp

```diff
@@ -121,6 +121,13 @@ void simplifyChildren(TR::Node * node, TR::Block * block, TR::Simplifier * s)
          child = s->simplify(child, block);
          node->setChild(i, child);
          }
+      // if simplification produced a PassThrough attach the child of the PassThrough here
```

Comment:
You may want to also check for GlRegDeps here since that's the other "valid" parent of a pass through node. Currently, I believe simplifier never runs after GRA and so this situation is probably never going to be encountered but it's still likely a good idea to future proof this code in case where simplifier is allowed to run changes.

Reply:
fixed in 31ed890

Reply:
Thanks, this looks ready to merge if the commit and PR message can be updated to reflect the GlRegDeps case that this is now handling.